README.md

```diff
@@ -10,12 +10,24 @@ inkjs is fully compatible with the original version, has zero dependency and wor
 
 ## Table of content
 
-- [Installation](#installation)
-- [Quickstart in the browser](#quickstart)
-- [Working with a JSON file](#working-with-a-json-file)
-- [Using Node.js](#using-nodejs)
-- [Differences with the C# API](#differences-with-the-c-api)
-- [Using the js Compiler](#compiler)
+- [](#)
```

Comment:
VSCode added indentation here, which seemed like a nice thing. Happy to remove though, if we want a cleaner PR.